### ISSUE TITLE
No javascript support

### ISSUE THREAD

**Issue by Jehan** [mrlem/cupa#12]
Cupa is basically about editing static pages. But all these pages get broken when javascript is disabled, which is a little contradictory.
It would be interesting if the html pages could be regenerated (keeping the original as a template only) with the new content. Therefore the page would be viewable for a user with javascript disabled as well.

I was interested by cupa, but it is not usable in the current state.
For instance a user with disability will only get an empty website.

**Comment by mrlem** [comment]
Indeed, that's a fair point, that I did dismiss in Cupa's current version. Sorry about that.

I'm thinking of some approaches to achieve this while retaining the page design simplicity (which makes it a little harder). For instance, use the noscript tag to redirect to a PHP file that would be able to create a static version of whatever page the user accessed (ideally, this redirect would have relied on the HTTP referer, but sadly that would depend on whether the browser is sending this header or not, plus that would prevent correct bookmarking support).

This feature will require a bit of work, but it's definitely worth it - thanks for the feedback :)
